### PR TITLE
[Feature] Add error layout + layout fullpage

### PR DESCRIPTION
**Pull request by cavasinf**
## Description
Add `error` layout page.

404: `/_error/404.html`
image

500: `/_error/500.html`
image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] I updated the documentation (see [here](https://github.com/kevinpapst/TablerBundle/tree/master/docs))
- [x] I agree that this code will be published under the [MIT license](https://github.com/kevinpapst/TablerBundle/blob/master/LICENSE)

### CODE REVIEW

**Comment by kevinpapst** [comment]
Thank you @cavasinf - works great!

I did some smaller refactorings and added [examples to the demo app](https://github.com/kevinpapst/TablerBundle-Demo/pull/5). It now uses the `status_code`  and `status_text` as fallback for missing translations:
Bildschirmfoto 2022-01-08 um 00 26 56

Can you adjust the French translation? It currently sounds as if it is targeted for 404 "page not found", but I think we should provide a generic error message instead. After that I will merge.

**Review by kevinpapst — CHANGES_REQUESTED** [review]
Please check French translation, I changed English and German yesterday. Everything els eis good to go from my side. But probably you wanna try my changes first?

**Comment by cavasinf** [comment]
Looks Good, but I'm not at my workspace rn.
Can only check next Monday.

File: translations/TablerBundle.fr.xliff [hunk 1]
```diff
@@ -94,6 +94,18 @@
                 <source>registration.confirmed</source>
                 <target>Félicitations %username%, votre compte est maintenant activé.</target>
             </trans-unit>
+            <trans-unit resname="http_error.title" id="Oops… You just found an error">
+                <source>Oops… You just found an error</source>
+                <target>Oups… Vous avez trouvé une page d'erreur</target>
+            </trans-unit>
+            <trans-unit resname="http_error.description" id="We are sorry but something went wrong">
+                <source>We are sorry but something went wrong</source>
+                <target>Nous sommes désolé, mais la page que vous cherchez n'a pas été trouvée</target>
```
Comment (cavasinf) [inline translations/TablerBundle.fr.xliff:103]:
```suggestion
                <target>Nous sommes désolés mais quelque chose s'est mal passé</target>
```

**Comment by kevinpapst** [comment]
Oh right: it is weekend, totally forgot 😁 thanks for contributing during your free time!

**Review by kevinpapst — APPROVED** [review]
Waiting with merge for your final tests!

**Comment by cavasinf** [comment]
@kevinpapst The "Take me home" trans doesn't work.
But I can't find why ...

image

**Comment by kevinpapst** [comment]
Just tested, works for me: 

Bildschirmfoto 2022-01-10 um 10 55 54
 
Bildschirmfoto 2022-01-10 um 10 56 14

Cache?

**Comment by cavasinf** [comment]
`php bin/console cache:clear` and `rm var/ -R` didn't change anything..

**Comment by kevinpapst** [comment]
I am testing in APP_ENV=prod and just did `rm -rf var/cache/*` after updating the branch.
Lets merge and see what happens then.

**Pull request merged**